diff -Nru indent-2.2.4/indent.texinfo indent-2.2.4.new/indent.texinfo
--- indent-2.2.4/indent.texinfo	Tue Jan 11 22:49:20 2000
+++ indent-2.2.4.new/indent.texinfo	Tue Jan 11 22:52:24 2000
@@ -15,10 +15,12 @@
 @setchapternewpage odd
 
 @ifinfo
-@dircategory Indent Code Formatter
+@format
+@dircategory Text Processing Tools:
 @direntry
-* Indent: (indent).		C code formatter.
+* indent: (indent).			Program to format C source code
 @end direntry
+@end format
 
 @c !BEGIN INTRO
 This is Edition @value{EDITION} of @cite{The @command{indent} Manual},
