diff -Nru indent-1.10.0/indent.texinfo indent-1.10.0.new/indent.texinfo
--- indent-1.10.0/indent.texinfo	Wed May 26 19:54:29 1999
+++ indent-1.10.0.new/indent.texinfo	Wed Jun 30 20:57:34 1999
@@ -14,6 +14,13 @@
 @synindex cp ky
 @setchapternewpage odd
 
+@format
+@dircategory File utilities:
+@direntry
+* indent: (indent).			Program to format source code
+@end direntry
+@end format
+
 @ifinfo
 @c !BEGIN INTRO
 This is Edition @value{EDITION} of @cite{The @code{indent} Manual},
