--- indent-2.2.8a/man/Makefile.am.orig	Tue May 21 21:00:58 2002
+++ indent-2.2.8a/man/Makefile.am	Sat Nov  9 00:11:58 2002
@@ -2,14 +2,14 @@
 
 bin_PROGRAMS = texinfo2man
 texinfo2man_SOURCES = texinfo2man.c
-man_MANS = @PACKAGE@.1
+man_MANS = $(PACKAGE).1
 
-EXTRA_DIST = $(man_MANS) @PACKAGE@.1.in
+EXTRA_DIST = $(man_MANS) $(PACKAGE).1.in
 
 # --------------- Maintainer's Section
 
-MAINTAINERCLEANFILES = @PACKAGE@.1 Makefile.in texinfo2man 
+MAINTAINERCLEANFILES = $(PACKAGE).1 Makefile.in texinfo2man 
 
-@PACKAGE@.1: ${srcdir}/@PACKAGE@.1.in  ${srcdir}/../doc/@PACKAGE@.texinfo texinfo2man  Makefile.am
-	./texinfo2man ${srcdir}/@PACKAGE@.1.in ${srcdir}/../doc/@PACKAGE@.texinfo > $@
+@PACKAGE@.1: $(srcdir)/$(PACKAGE).1.in  ../doc/$(PACKAGE).texinfo texinfo2man  Makefile.am
+	./texinfo2man $(srcdir)/$(PACKAGE).1.in ../doc/$(PACKAGE).texinfo > $@
 
