--- indent-2.2.10/doc/Makefile.am~	2008-03-11 18:50:42.000000000 +0000
+++ indent-2.2.10/doc/Makefile.am	2014-12-28 13:59:25.881825789 +0000
@@ -34,7 +34,7 @@
 html-split: @PACKAGE@_toc.html
 
 @PACKAGE@.html: version.texi $(@PACKAGE@_TEXINFOS)
-	$(TEXI2HTML) -expandinfo -number -monolithic `if test -f @PACKAGE@.texinfo; then echo @PACKAGE@.texinfo; else echo $(srcdir)/@PACKAGE@.texinfo; fi`
+	$(TEXI2HTML) -expandinfo -number-footnotes -number-sections -monolithic `if test -f @PACKAGE@.texinfo; then echo @PACKAGE@.texinfo; else echo $(srcdir)/@PACKAGE@.texinfo; fi`
 
 @PACKAGE@_toc.html: version.texi $(@PACKAGE@_TEXINFOS)
 	case "$(TEXI2HTML)" in \
